pkgs/applications/networking/mailreaders/claws-mail/default.nix

```diff
@@ -41,18 +41,19 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
-  patches = [ ./mime.patch ];
+  patches = [ ./mime.patch ./disable-python-detection.patch ];
```

Comment:
What happens without the patch?

Reply:
Without the patch, it checks for python and tries to dlopen libpython27.so during `configure` and fails, therefore disables the python plugin.

Without this check, it still compiles and works fine. The better way might be fixing the check, but on the other hand, we *know* python will be there, since it's a dependency.

Reply:
I can provide you the config.log, if you want to. The relevant part looks something like:
```
| #include <dlfcn.h>
|                    #define PYTHON_SO_FILE "libpython2.7.so"
|
| int
| main ()
| {
| if (!dlopen(PYTHON_SO_FILE, RTLD_NOW | RTLD_GLOBAL)) return 1; return 0;
|   ;
|   return 0;
| }
|
configure:22304: result: no
configure:22306: WARNING: Could not find Python shared libary: libpython2.7.so. Maybe you need to install development packages for Python.```

Reply:
Ok then the following snippet will be slightly shorter and do not require to maintain a patch:


```
+
+  preConfigure = ''
+    # autotools tries to dlopen libpython to check as a requirement to build the
+    # python plugin
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${python}/lib
+  '';
+
```

Reply:
Done. Thanks for all your input! This is definitely cleaner.